README.md

```diff
@@ -171,5 +171,5 @@ To check if your fetch request is working, you can look at the `db.json` file to
 
 **YOUR NOTES**
 ```
-
+first figured it out without persisting. then added persisting. used a post request because creating a new goal. used a different api url to get to the goals. first 2 videos i did by manually entering the id. the last one, commented out the id and it persisted. how do you persist with the id??
```

Comment:
The server will generate the ID, so it's not something you need to send with the request. Think about how it would work when creating a new row in a database table - the ID is something the database creates for you, not something you need to set manually.